### ISSUE TITLE
strips from audacity seems to have some gain applied

### ISSUE THREAD

**Issue by samytichadou** [tin2tin/audacity_tools_for_blender#16]
When sending a single strip from blender to audacity, if user does not do anything to it, and get it back into blender, it seems (according to the waveform), that this strip audio has been levelled up in audacity. 
You can see it in this screenshot (the up one is from audacity, the lower one is the source)

![image](https://user-images.githubusercontent.com/26276003/111320577-6a69bc80-8667-11eb-9669-c0f955505a66.png)

**Comment by tin2tin** [comment]
Yes, I noticed this too.  Both in and out of Audacity, there is something going on with the levels. I've added envelope points for the strips being send to Audacity, I guess we could also do this for files returning, but it's hard to know how much. 

Maybe it's because Audacity is handling the audio logarithmic and the VSE linear, but I do not know anything about these things.

Also there might be inaccuracies in the waveform drawing in Blender. Try to extend or give height to a strip and see how much they're jumping, also in shape.

**Comment by samytichadou** [comment]
> Try to extend or give height to a strip and see how much they're jumping, also in shape.

Problem seems to be in the files, consistent waveform drawing
![image](https://user-images.githubusercontent.com/26276003/111325418-f8e03d00-866b-11eb-92d5-94d4981d97f4.png)

And after increasing volume value of both
![image](https://user-images.githubusercontent.com/26276003/111325734-42308c80-866c-11eb-9730-5896213ba6da.png)

**Comment by tin2tin** [comment]
Yes, that's a clear argument of the volume has been gained in Audacity.

**Comment by tin2tin** [comment]
I asked about this here: https://forum.audacityteam.org/viewtopic.php?p=419781#p419781

**Comment by tin2tin** [comment]
In Audacity all audio will be placed in stereo tracks, because I don't know how to detect if the source audio in Blender is mono or not, and in Audacity it also needs to be placed in stereo tracks because other files in that track may need to be stereo. 

I suspect the additional volume may be caused by the process from going from mono to stereo and then to a downmix/export, but I don't know enough about what happens in Audacity during that process.
